### ISSUE TITLE
[BUG] 3.0.5 is incompatible with Illuminate\Contracts\Console\Kernel

### ISSUE THREAD

**Issue by nebulade** [Leantime/leantime#2293]
While testing the Cloudron package update to 3.0.5 our tests found a change which makes the function signature of https://github.com/Leantime/leantime/commit/8d6fb112fc6492ce0018b299a4a1923bfde83973#diff-6c062770a1419092e7e7afa83d730d3ab663ec996f669f546c44db15a7af5ae2R241 incompatible with Illuminate.

```
[08-Feb-2024 16:16:05 UTC] PHP Fatal error:  Declaration of Leantime\Core\ConsoleKernel::queue(string $command, array $parameters = []) must be compatible with Illuminate\Contracts\Console\Kernel::queue($command, array $parameters = []) in /app/code/app/Core/ConsoleKernel.php on line 241
```

If the `string` is removed from that line, the app starts working again. I could make a PR for this, but since I am no php developer I am not sure what the implications of this are.

**Comment by marcelfolaron** [comment]
Hey,

You are correct. I have a fix in master and will create a new release here shortly.

**Comment by marcelfolaron** [comment]
Fixed in 3.0.6
